websites/P/Paramount+/presence.ts

```diff
@@ -0,0 +1,235 @@
+const presence = new Presence({
+  clientId: "821433038335377418"
+});
+(async function () {
+  const strings = await presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    live: "presence.activity.live",
+    search: "presence.activity.searching"
+  });
+
+  let title: string,
+    seasonEpi: string,
+    movTitle: string,
+    vidArea: Element,
+    vidMdTl: Element,
+    elapsed: number = undefined,
+    oldUrl: string = undefined;
+
+  presence.on("UpdateData", () => {
+
+    vidArea = document.querySelector(".video__player-area");
+    let video: HTMLVideoElement = null,
+      details = undefined,
+      state = undefined,
+      smallImageKey = undefined,
+      smallImageText = undefined,
+      startTimestamp = undefined,
+      endTimestamp = undefined;
```

Comment:
just do presenceData object here and just assign things or remove them by ``delete``